OpenEdXMobile/build.gradle

```diff
@@ -510,6 +521,22 @@ android.applicationVariants.all { variant ->
         tasks.getByName(googleServicesJsonValidationTask).dependsOn(serviceJsonTask)
     }
 
+    // Task that generate auth_config.json for the config and execute for the Microsoft authentication
```

Comment:
The task that generates auth_config.json for the config and executes for the Microsoft authentication

Reply:
Done.